### PR TITLE
ci: updates to kata setup scripts for ppc64le

### PR DESCRIPTION
**Pull request by jing-wang4**
- kubenetes package installation for ppc64le
- virtiofsd installation

Fixes: #3185
Signed-off-by: Jing Wang <jing.wang4@ibm.com>

### CODE REVIEW

**Comment by jing-wang4** [comment]
@wainersm 
This is a follow-up for the [PR](https://github.com/kata-containers/tests/pull/3205) because of merge conflicts. please review. Thanks!

File: .ci/install_kubernetes.sh [hunk 1]
```diff
@@ -29,10 +30,12 @@ EOF"
 	chronic sudo -E apt update
 	chronic sudo -E apt install --allow-downgrades -y kubelet="$kubernetes_version" kubeadm="$kubernetes_version" kubectl="$kubernetes_version"
 elif [ "$ID" == "centos" ] || [ "$ID" == "fedora" ]; then
-	sudo bash -c "cat <<EOF > /etc/yum.repos.d/kubernetes.repo
+	url="https://packages.cloud.google.com/yum/repos/kubernetes-el7-${ARCH}"
+        echo "Install ${url} for ${ARCH}"
```
Comment (chavafg) [inline .ci/install_kubernetes.sh:34]: can you please fix indentation?

Comment (jing-wang4) [reply .ci/install_kubernetes.sh:34]: @chavafg ,
Somehow I am not seeing these indentation at my local, also viewing the [file](https://github.com/kata-containers/tests/blob/641bbe8a8d8f5fc5a7e028b3b47b57db3c6655f1/.ci/install_kubernetes.sh#L34), looks good too. Not sure what missed here. thanks!

Comment (chavafg) [reply .ci/install_kubernetes.sh:34]: @jing-wang4, hi, seems that you are using spaces and the other lines in the file are using tabs

Comment (jing-wang4) [reply .ci/install_kubernetes.sh:34]: Thanks @chavafg !
I mistakenly got my fork deleted, so please review the duplicate/new [PR](https://github.com/kata-containers/tests/pull/3244) and close this one. Thanks!

**Comment by jing-wang4** [comment]
close this one, please review the duplicate/new [PR](https://github.com/kata-containers/tests/pull/3244).

**Pull request closed without merging**